go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/gookit/color v1.2.7
 	github.com/hashicorp/go-multierror v1.1.0
 	github.com/hashicorp/go-version v1.2.0
+	github.com/k0kubun/go-ansi v0.0.0-20180517002512-3bf9e2903213
```

Comment:
Just double-checking that this was intended, since I couldn't remember if we're adding new ANSI logic here.

Reply:
I've used this package before and decided to use it for the hide/show of the cursor (was intended)